Improvement:  ik clavicle now builds on joints that aren't aligned to world.

pumacamargo opened the pull request:
My shoulders were not worldspace oriented, so when the code freeze transformation in line 85, it was setting the "clavicle_rotateLock_" to a worldspace orietation that was causing the locator not matching with the position of the pole vector and flipping the shoulder

By add a xform group instead of the freeze transformation, it works in any orientation.

pumacamargo committed:
Update ik_clavicle.py

My shoulders were not worldspace oriented, so when the code freeze transformation in line 85, it was setting the "clavicle_rotateLock_" to a worldspace orietation that was causing the locator not matching with the position of the pole vector and flipping the shoulder

By add a xform group instead of the freeze transformation, it works in any orientation.

louisVottero commented on character/human/body/.code/rig/arms/ik_clavicle/ik_clavicle.py:
```diff
@@ -78,21 +78,19 @@ def main():
         
         #fix pole vector
         loc = cmds.spaceLocator(n = 'clavicle_rotateLock_%s' % side)[0]
+        locXform = cmds.group(loc ,n = 'xform_clavicle_rotateLock_%s' % side)
```
use space.create_xform_group(loc) instead of cmds.group(...)
this will handle the xform_ naming, and if there were an offset it would automatically add it.

louisVottero replied:
locXform should follow the python pep8 rules
something like loc_xform instead of camelCase

louisVottero reviewed (changes requested):
I've added some notes with minor changes

pumacamargo committed:
Improvement: ik clavicle pole vector update

This will help the pole vector to work in any orientation, not only in world space
